src/CodeGen_LLVM.cpp

```diff
@@ -603,6 +608,12 @@ void CodeGen_LLVM::begin_func(LoweredFunc::LinkageType linkage, const std::strin
     BasicBlock *block = BasicBlock::Create(*context, "entry", function);
     builder->SetInsertPoint(block);
 
+    // Put the global buffers in the symbol table.
+    for (const auto &buf : input_module->buffers()) {
+        llvm::Value *buf_ptr = sym_get(buf.name() + ".buffer");
```

Comment:
This LGTM either way. It would be nice to eliminate any inconsistency between global buffers and parameters. Maybe we should do it later since it is already inconsistent.